From 3f48602ab837e43fbcdef3866eb48a403164cd7e Mon Sep 17 00:00:00 2001
From: terminills <tdeters@gmail.com>
Date: Fri, 24 Jul 2026 16:39:52 -0500
Subject: [PATCH 13/43] cef: include map for crash key registry

Declare CefCrashReporterClient's std::map dependency directly instead of relying on a transitive standard-library include.

AI_PHASE: chromium-cef-bootstrap

AI_STATUS: gcc-direct-include-corrected

AI_VALIDATION: build 20 reports std::map is undefined at crash_reporter_client.h:91

AI_EVIDENCE: ../test-artifacts/-cef-bootstrap-20260724/20-libcef-static-handoff-boundary.log

AI_DECISION_CHAIN:; 6247ab535b; 7b48d030
---
 libcef/common/crash_reporter_client.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libcef/common/crash_reporter_client.h b/libcef/common/crash_reporter_client.h
index 2fd9c73d..94de66f2 100644
--- a/libcef/common/crash_reporter_client.h
+++ b/libcef/common/crash_reporter_client.h
@@ -5,6 +5,7 @@
 #ifndef CEF_LIBCEF_COMMON_CRASH_REPORTER_CLIENT_H_
 #define CEF_LIBCEF_COMMON_CRASH_REPORTER_CLIENT_H_
 
+#include <map>
 #include <string>
 #include <vector>
 
